install/set_gigabyte_node_bmc_to_factory_defaults.md

```diff
@@ -1,111 +1,125 @@
 # Set Gigabyte Node BMC to Factory Defaults
 
-### Prerequisites
+There are cases when a Gigabyte node BMC must be reset to its factory default settings. This page describes when this reset is appropriate, and
+how to use management scripts and text files to do the reset.
 
-Use the management scripts and text files to reset Gigabyte BMC to factory default settings. Set the BMC to the factory default settings in the following cases:
+Set the BMC to the factory default settings in the following cases:
 
-- There are problems using the ipmitool command and Redfish does not respond
-- There are problems using the ipmitool command and Redfish is running
-- When BIOS or BMC flash procedures fail using Redfish
-  - Run the `do_bmc_factory_default.sh` script
-  - Run `ipmitool -I lanplus -U admin -P password -H BMC_or_CMC_IP mc reset cold` and flash it again after 5 minutes seconds
-- If booted from the PIT node:
-  - the firmware packages are located in the HFP package provided with the Shasta release
-  - the required scripts are located in `/var/www/fw/river/sh-svr-scripts`
+- There are problems using the `ipmitool` command and Redfish does not respond.
+- There are problems using the `ipmitool` command and Redfish is running.
+- When BIOS or BMC flash procedures fail using Redfish.
 
-### Procedure
+## Procedure
 
-#### Apply the BMC Factory Command
+**This section refers to scripts that exist only in the PIT environment.** If necessary, copy the LiveCD data from a different machine to get these scripts.
+
+**Note**: When BIOS or BMC flash procedures fail using Redfish:
+
+- Run the `do_bmc_factory_default.sh` script
+- Run `ipmitool -I lanplus -U admin -P password -H BMC_or_CMC_IP mc reset cold` and flash it again after 5 minutes (300 seconds).
+
+If booted from the PIT node:
+
+- The firmware packages are located in the HPE Cray EX HPC Firmware Pack (HFP) provided with the Shasta release.
+- The required scripts are located in `/var/www/fw/river/sh-svr-scripts`
+
+### Apply the BMC factory command
 
 1. Create a `node.txt` file and add the target node information as shown:
 
-   Example `node.txt` file with two nodes:
+    Example `node.txt` file with two nodes:
 
-   ```screen
+    ```text
     10.254.1.11 x3000c0s9b0 ncn-w002
     10.254.1.21 x3000c0s27b0 uan01
-   ```
+    ```
 
    Example `node.txt` file with one node:
 
-   ```screen
+    ```screen
     10.254.1.11 x3000c0s9b0 ncn-w002
-   ```
+    ```
 
 2. Use Redfish to reset the BMC to factory default.
 
-   The BMC is running 12.84.01 or a later version.
+    - **Option 1:** If the BMC is running version `12.84.01` or later, then run:
+
+        ```bash
+        ncn# sh do_Redfish_BMC_Factory.sh
+        ```
+
+    - **Option 2:** Use `ipmitool` to reset the BMC to factory defaults:
 
-   ```bash
-   ncn-w001# sh do_Redfish_BMC_Factory.sh
-   ```
+        ```bash
+        ncn-w001# sh do_bmc_factory_default.sh
```

Comment:
Nothing about this procedure indicates that it must be run on ncn-w001 specifically. This is likely a holdover from the old days when many tasks were done from there. I don't see why it would be required in this case.

```suggestion
        ncn# sh do_bmc_factory_default.sh
```